### DISCUSSION TITLE
Allow sorting of branches/tags by last modified

### DISCUSSION THREAD

**Discussion by kerryj89** [community/community#34123] [Repositories]
Please allow us to sort by last modified.  Search functionality already exists so alphabetization is less important, and main branch is already pinned so it doesn't get lost.

![image](https://user-images.githubusercontent.com/1977804/192154859-f4736894-3e75-4852-932b-58052db6e3b6.png)

**Comment by h-vetinari** [comment]
OMG please yes! It's so annoying to see ancient branches first and have to go hunting for the last release (especially for repos that only have main/master + a given naming scheme for release branches). It's also inconsistent with tags, which _are_ sorted in reverse order.
